### PR TITLE
v1 auth

### PR DESCRIPTION
**Pull request by TheBlueSummon**
Having trouble with Auth context in ./client/routes/__root.tsx
I should have an auth route already that should like to website/authed
Would be very helpful if someone can look over it and lmk what went wrong

### CODE REVIEW

File: client/routes/About.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { createFileRoute } from '@tanstack/react-router'
 
-export const Route = createFileRoute('/about')({
+export const Route = createFileRoute('/About')({
```
Comment (ethanniser) [inline client/routes/About.tsx:3]: this should probably be undone...

File: client/routes/Profile.tsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { createFileRoute } from '@tanstack/react-router'
 
-export const Route = createFileRoute('/profile')({
+export const Route = createFileRoute('/Profile')({
```
Comment (ethanniser) [inline client/routes/Profile.tsx:3]: same thing

**Review by ethanniser — CHANGES_REQUESTED** [review]
https://tanstack.com/router/latest/docs/framework/react/guide/authenticated-routes#redirecting

lets try doing somehting like this is the `routes/_authenticated.tsx` layout

then just put all authenticated routes under `routes/_authenticated/<route>.tsx`

**Comment by ufsasewebmaster** [comment]
Closed because of updated #77. Pretty sure there was a misuderstanding of what files where what when adding the example code (+ branched from main, not sprint-2, outdated)

**Pull request closed without merging**